### ISSUE TITLE
The results of S3DIS

### ISSUE THREAD

**Issue by fxy2012** [Yang7879/3D-BoNet#19]
After testing, I got some .mat files. And there are many blocks in them. But I don't know how to deal with them to get the visualization of them. How should I put them together?

**Comment by zhaoyu19920930** [comment]
Hi, Yang7879, I have the same confusion as fxy2012, after I run the main_train.py code or the main_eval.py code, it will create "eval scene Area_5_WC_1.h5.mat" in my log file, but I don't konw what's the .h5.mat files?Or how to deal with it?
Could you please explain the .h5.mat files. Thanks.

**Comment by Yang7879** [comment]
Hi @fxy2012  and @zhaoyu19920930 ,  the '.mat' file includes all the predicted semantic and instance results together with ground truth for each scene (room). Refer this line below to check how the data are organized within '.mat' file. https://github.com/Yang7879/3D-BoNet/blob/b4345f28554808f9c07b42bb5487df48c859d89f/main_eval.py#L157

To visualize the results, simply uncomment the lines below:
https://github.com/Yang7879/3D-BoNet/blob/b4345f28554808f9c07b42bb5487df48c859d89f/main_eval.py#L212

Basically, for each scene(room), the results are predicted per block and stored in '.mat' file. Before the final evaluation, those blocks are merged together and then the entire room is available for visualization.  

Btw, the '.h5.mat' file is a '.mat' file, nothing to do with '.h5' format, it's just the file name.
